.github/workflows/android.yml

```diff
@@ -0,0 +1,150 @@
+name: Android CI
+
+on:
+  push:
+    branches:
+      - master
+      - '!gh-pages'
+    paths:
+      # Rebuild when workflow configs change.
+      - .github/workflows/**
```

Comment:
Be more specific: only consider this actual file. No need to rebuild Android when iOS config changes.